jenkins/release_ci/opensearch_maven_release/maven-sign-release.jenkinsfile

```diff
@@ -0,0 +1,94 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            defaultValue: 'x64',
+            name: 'ARCH',
```

Comment:
Unrelated thought @peternied @zelinh - do we really need this argument (`ARCH` and `PLATFORM`), given they are only used as folder paths?

Reply:
Are you saying we could combine `ARCH` and `PLATFORM` to one parameter like called `relative path`?

Reply:
We could remove this parameter altogether since the value is fixed.